packages/trace-viewer/src/ui/workbenchLoader.tsx

```diff
@@ -157,6 +157,7 @@ export const WorkbenchLoader: React.FunctionComponent<{
       <button onClick={() => {
         const input = document.createElement('input');
         input.type = 'file';
+        input.multiple = true;
         input.click();
         input.addEventListener('change', e => handleFileInputChange(e));
       }}>Select file</button>
```

Comment:
should the text on this button be updated too?
```suggestion
      }}>Select file(s)</button>
```

Reply:
```
      }}>Select file(s) (That's right, you can select multiple files at once and they will be smushed into a single trace!)</button>
```
?